### PR TITLE
Add endpoints for episodes

### PR DESCRIPTION
**Pull request by Rezz888**
# Description

- What work was done?
   1. Full CRUD functionality of the Story and the Episode endpoint.
   2. All tests are written for the above as well.
- Why was this work done?
  This is done to support the newly structured New Story and Episodes schema and for it to have a fully functional API's. The aim is to use this in production eventually once approved, replacing the old story which is currently in use now.
- What feature is it for?
  New story and episodes data schema
- Any relevant links or images / screenshots
  https://trello.com/b/aZl8089B/scribble-stadium-schema

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

## Has This Been Tested

- [x] Yes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: api/stories/newStoryRouter.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ router.get('/:id', (req, res) => {
 });
```
Comment (jayaramsivaramannair) [inline api/stories/newStoryRouter.js]: You need to run the lint script to address these formatting errors.

**Review by jayaramsivaramannair — DISMISSED** [review]
A simple yet important piece of code.  Good work.

**Comment by Rezz888** [comment]
> A simple yet important piece of code. Good work.

Thank you! It is still in draft mode. We're running some issues with the tests. Will finalize once this is sorted.

File: api/stories/newStoryRouter.js (hunk 1 above)
Comment (Rezz888) [reply api/stories/newStoryRouter.js]: Thanks for your feedback. Will do so asap

Comment (jayaramsivaramannair) [reply api/stories/newStoryRouter.js]: Yes that will be a quick fix.

File: api/stories/storyNewRouter.js [hunk 2]
```diff
@@ -62,41 +61,47 @@ router.get('/episodes/:eid', async(req, res, next) => {
...
   } catch (error) {
     res.status(404).json({
-      message: "EpisodeNotFound",
-    })
+      message: 'EpisodeNotFound',
+    });
   }
 });
 
 //Work in progress by next cohort to add constraint to EpisodeNumber to be unique per story ID
```
Comment (jayaramsivaramannair) [inline api/stories/storyNewRouter.js:74]: Good use of comments throughout the code.

**Review by jayaramsivaramannair — APPROVED** [review]

File: api/stories/storyNewRouter.js (hunk 2 above)
Comment (allraec) [reply api/stories/storyNewRouter.js:74]: Thanks!

**Review by kwnie — APPROVED** [review]
Great job pushing through and writing those tests, I know there were some roadblocks there. The code is clean and reads well, good work!

**Pull request merged**